source/documentation/requirements.html.md

```diff
@@ -80,42 +83,106 @@ the CloudSIG using rdopkg:
           cloud7-openstack-ocata-release: python-eventlet-0.18.4-2.el7
           cloud7-openstack-ocata-testing: python-eventlet-0.18.4-2.el7
           cloud7-openstack-pike-testing: python-eventlet-0.20.1-2.el7
+          cloud7-openstack-pike-release: python-eventlet-0.20.1-2.el7
+          cloud7-openstack-queens-release: python-eventlet-0.20.1-2.el7
+          cloud7-openstack-queens-testing: python-eventlet-0.20.1-2.el7
+          cloud7-openstack-rocky-testing: python-eventlet-0.20.1-2.el7
+
         master-distgit: https://github.com/rdo-common/python-eventlet.git
         review-origin: null
         review-patches: null
         tags:
           dependency: null
-        maintainers: 
+        maintainers:
         - apevec@redhat.com
         - hguemar@fedoraproject.org
 
-    If the package is found, it's already in RDO repositories and no more actions are needed to add
-    it to the repos.
-    
+    Note that the version of the package included in repositories is given by the
+    CBS tags applied to each package (shown under buildsys-tags section for each package).
+    Tags have a format cloud7-openstack-&lt;release>-&lt;phase> where:
+
+    - release: is a the OpenStack release name, as pike, queens or rocky.
+    - phase:
+      - `candidate` phase is assigned to packages to be rebuilt in
+      CBS but not pushed to any RDO repository.
+      - `el7-build` (only available for Rocky and newer releases)
+      is assigned to packages that only required to build other
+      packages but not to deploy or run any OpenStack service.
```

Comment:
but are not a runtime requirement for any other package (it looks a bit confusing).

Reply:
fixed